### PR TITLE
TCVP-1680 Integration with ORDS

### PR DESCRIPTION
**Pull request by karahanb**
# Description

This PR includes the following proposed change(s):

- [TCVP-1680](https://justice.gov.bc.ca/jira/browse/TCVP-1680)
- Added Spring WebClient configuration with timeout and retry in order to perform communications with ORDS.
- Added functionality to get statutes from the ORDS statutes endpoint to save into Redis.
- Updated docker-compose and helm chart with the new ORDS connection related environment variables.
- Updated 'Statute' object in Common project to match the json returned from ORDS and saved in Redis. Also, updated StatuteLookupService's section based query as well.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Ran all existing and new unit tests. Ran the project locally and sent a GET request to the "/api/v1.0/codetable/refresh" to trigger refreshing statutes from the ORDS endpoint and verified that the results has been saved to Redis successfully. 

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/Statute.java [hunk 1]
```diff
@@ -14,19 +11,25 @@
 @Getter
 @Setter
 @NoArgsConstructor
+@AllArgsConstructor
 public class Statute {
```
Comment (pbolduc) [inline src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/Statute.java:15]: When we pull in the ORDS Statue, can we map to our column names. For example this seems cleaner

public record Statute(
    string Id, 
    string ActCode, 
    string SectionText, 
    string SubsectionText,
    string ParagraphText, 
    string SubparagraphText, 
    string Code, 
    string ShortDescriptionText, 
    string DescriptionText
);

File: src/backend/TrafficCourts/Common/Models/Statute.cs [hunk 2]
```diff
@@ -1,3 +1,4 @@
 ﻿namespace TrafficCourts.Common.Models;
 
-public record Statute(decimal Code, string Act, string Section, string Description);
+public record Statute(string StatId, string ActCd, string StatSectionTxt, string StatSubSectionTxt,
```
Comment (pbolduc) [inline src/backend/TrafficCourts/Common/Models/Statute.cs:3]: Can we map to our cleaner names like

public record Statute( string Id, string ActCode, string SectionText, string SubsectionText, string ParagraphText, string SubparagraphText, string Code, string ShortDescriptionText, string DescriptionText );

Comment (karahanb) [reply src/backend/TrafficCourts/Common/Models/Statute.cs]: Now we have a mapper and a DTO class in Oracle Data API to map statute JSON from ORDS to a StatuteDTO utilizing Mapstruct library to save the object with cleaner names as you mentioned above for the use of other projects.

**Review by pbolduc — APPROVED** [review]
Looks good now

**Pull request merged**